package.json

```diff
@@ -79,7 +79,7 @@
     "gulp-mocha": "^2.0.0",
     "gulp-shell": "^0.5.2",
     "mochainon": "^2.0.0",
-    "pkg": "^4.3.0-beta.1",
+    "pkg": "4.3.8",
```

Comment:
Because older CLI versions are tested with Node 6, and 4.3.8 is the highest `pkg` version that maintains Node 6 compatibility.